index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Dammy's Portfolio</title>
+ <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+ <!-- Header Section -->
+
+ <header>
+  <div id="page_logo">
+   <span>Dammy Shittu</span>
+  </div>
+
+  <nav>
+   <img src="./img/union.png" alt="Hamburger Image">
+  </nav>
+
+ </header>
+
+ <!-- End of Header Section -->
+
+ <!-- Main Section -->
+ <main >
+  <section class="container">
+   <div id="circles">
+    <img src="./img/ellipse-1.svg" class="circle-1" alt="Circle 1">
+    <img src="./img/ellipse-2.svg" class="circle-2" alt="Circle 2">
+   </div>
```

Comment:
- [ ] Please try to add a proper background image as followed in the `Figma` design. The decorations are always background image, not `img` tags.